app/models/episode.rb

```diff
@@ -57,6 +57,8 @@ class Episode < ApplicationRecord
 
   scope :published_by, ->(offset) { where("published_at IS NOT NULL AND published_at <= ?", Time.now + offset) }
 
+  scope :scheduled, -> { where("released_at > ?", Time.now) }
```

Comment:
Bit of nuance in these labels, but following the Publish/CMS logic...

- An episode is "draft" when `published_at` is nil. But a draft can still have a dropdate set, with a non-nil `released_at`
- An episode is "scheduled" when we copy `released_at -> published_at`.  (And we also verify the episode has all its audio and is "ready" to publish, etc).  It's only scheduled while that date is > Time.now.
- An episode is "published" when we reach that `published_at` timestamp

So here, I think you want to check the `published_at` field instead?  And you'll also need to check that it's not null - in fact, you may want to just copy the `scope :published` and modify that.